synapse/http/site.py

```diff
@@ -225,7 +225,7 @@ def _started_processing(self, servlet_name):
             self.start_time, name=servlet_name, method=self.get_method()
         )
 
-        self.site.access_logger.info(
+        self.site.access_logger.debug(
```

Comment:
I'd expect inbound traffic to be logged at `INFO` but ymmv.

Reply:
we log when it completes at INFO. I think this is the right choice.

Reply:
Fair enough.